pimd/pim_vxlan.h

```diff
@@ -148,4 +148,6 @@ extern bool pim_vxlan_do_mlag_reg(void);
 extern void pim_vxlan_inherit_mlag_flags(struct pim_instance *pim,
 		struct pim_upstream *up, bool inherit);
 
+// Shutdown of PIM stop the thread
```

Comment:
do we use this kind of comment style?

Reply:
fixed, anything else needed @ton31337  ?